internal/charmstore/search_test.go

```diff
@@ -394,6 +394,55 @@ var searchTests = []struct {
 			exportTestCharms["varnish"],
 			exportTestBundles["wordpress-simple"],
 		},
+	}, {
+		about: "blank owner filter search",
+		sp: SearchParams{
+			Text: "",
+			Filters: map[string][]string{
+				"owner": {""},
+			},
+		},
+		results: []*router.ResolvedURL{
+			exportTestCharms["wordpress"],
+			exportTestCharms["mysql"],
+			exportTestBundles["wordpress-simple"],
+		},
+	}, {
+		about: "promulgated search",
+		sp: SearchParams{
+			Text: "",
+			Filters: map[string][]string{
+				"promulgated": {"1"},
+			},
+		},
+		results: []*router.ResolvedURL{
+			exportTestCharms["wordpress"],
+			exportTestCharms["mysql"],
+			exportTestBundles["wordpress-simple"],
+		},
```

Comment:
Could we add test that owner= and promulgated=1 return the same results? We must rely on keeping owner= around for now.